show-land/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WeProDev BeTheme</title>
+
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+
+   <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+    <div class="background-color">
+
+        <div class="container">
+            <header class="header ">
+                <h1 class="logo">Anvogue</h1>
+                <nav class="main-nav">
+                    <ul class="main-nav__list">
+                        <li> <a class="main-nav__link" href="#">demos</a> </li>
+            <li><a class="main-nav__link" href="#">mobile view</a> </li>
+            <li><a class="main-nav__link" href="#">features</a> </li>
+            <li><a class="main-nav__link" href="#">help & support</a> </li>
+            <li><a class="main-nav__link btn btn-buttom nav-cta" href="#">purchase now</a> </li>
+        </ul>
+    </nav>
+</header>          
+</header>
```

Comment:
You can create your own header like this code
```
   <div class="header">
      <div class="header__main">
        <div class="header__logo">Anvogue</div>

        <nav class="header__nav">
          <ul>
            <li>
              <a href="#"><i class="fa-solid fa-bars-staggered"></i></a>
            </li>
            <li><a href="#">demos</a></li>
            <li><a href="#">mobile view</a></li>
            <li><a href="#">features</a></li>
            <li><a href="#">help & support</a></li>
          </ul>
        </nav>

        <div class="cta__button">
          <button class="cta__button--button">purchase now</button>
        </div>
      </div>
    </div>
```
It is very useful and beautiful
BEM rules are also used in this code